yellowbrick/classifier/boundaries.py

```diff
@@ -0,0 +1,457 @@
+            the meshgrid will be 0.25%% of differenes of the max and min of x
+            and y for each feature.
+
+        markers : iterable of strings, default: ,od*vh+
+            Matplotlib style markers for points on the scatter plot points
+
+        pcolormesh_alpha : float, default: 0.8
+            Sets the alpha transparency for the meshgrid of model boundaries
+
+        scatter_alpha : float, default: 1.0
+            Sets the alpha transparency for the scatter plot points
+
+        title : string, default: stringified feature_one and feature_two
+            Sets the title of the visualization
+
+        kwargs : keyword arguments passed to the super class.
+
+        These parameters can be influenced later on in the visualization
+        process, but can and should be set as early as possible.
+
+    """
+
+    def __init__(self,
+                 model,
+                 x=None,
+                 y=None,
+                 colors=None,
+                 classes=None,
+                 features=None,
+                 show_scatter=True,
+                 step_size=0.0025,
+                 markers=None,
+                 pcolormesh_alpha=0.8,
+                 scatter_alpha=1.0,
+                 title=None,
+                 **kwargs):
+        """
+        Pass in a unfitted model to generate a decision boundaries
+        visualization.
+        """
+        super(DecisionBoundariesVisualizer, self).__init__(self)
+
+        self.x = x
+        self.y = y
+        self.colors = colors if colors is not None else PALETTES['set1']
+        self.classes_ = classes
+        self.features_ = features
+        self.estimator = model
+        self.name = get_model_name(self.estimator)
+        self.show_scatter = show_scatter
+        self.step_size = step_size
+        self.markers = itertools.cycle(
+            kwargs.pop('markers', (',', 'o', 'd', '*', 'v', 'h', '+')))
+        self.pcolormesh_alpha = pcolormesh_alpha
+        self.scatter_alpha = scatter_alpha
+        self.title = title
+
+        # these are set later
+        self.Z = None
+        self.Z_shape = None
+        self.xx = None
+        self.yy = None
+        self.class_labels = None
+
+        if self.x is not None and self.y is not None and self.features_ is not None:
+            raise YellowbrickValueError(
+                'Please specify x,y or features, not both.')
+
+        if self.x is not None and self.y is not None and self.features_ is None:
+            self.features_ = [self.x, self.y]
+
+        # Ensure with init that features doesn't have more than two features
+        if features is not None:
+            if len(features) != 2:
+                raise YellowbrickValueError(
+                    'DecisionBoundariesVisualizer only accepts two features.')
+
+    def _select_feature_columns(self, X):
+        """ """
+
+        if len(X.shape) == 1:
+            X_flat = X.view(np.float64).reshape(len(X), -1)
```

Comment:
X.view has been giving me some trouble; apparently a future version of numpy is going to change it's behavior. Are you getting any warnings when using this? If not, would you mind `pip install -U numpy` and seeing if you get any warnings after? I've been trying to figure it out, but no luck yet. It may just have to do with named arrays.